linux/LinuxProcessList.c

```diff
@@ -1366,11 +1366,12 @@ static char* LinuxProcessList_updateTtyDevice(TtyDriver* ttyDrivers, unsigned lo
    return out;
 }
 
-static bool LinuxProcessList_recurseProcTree(LinuxProcessList* this, openat_arg_t parentFd, const char* dirname, const Process* parent, double period) {
+static bool LinuxProcessList_recurseProcTree(LinuxProcessList* this, openat_arg_t parentFd, const char* dirname, const LinuxProcess* lparent, double period) {
    ProcessList* pl = (ProcessList*) this;
    const struct dirent* entry;
    const Settings* settings = pl->settings;
    const ScreenSettings* ss = settings->ss;
+   const Process *parent = (const Process *) lparent;
```

Comment:
Style nit-picking: `const Process* parent = (const Process*) lparent;`

Reply:
Sure, amended and pushed.

Reply:
Not fully, the left side also has the `*` aligned on the type …

Reply:
Sorry, I should have noticed, fixed.